annotate.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+if [[ -z "${OPENAI_API_KEY}" ]]; then
+  echo "The openai api key env var is not set, please set a value and reinvoke"
+  exit 1
+
```

Comment:
```suggestion
fi
```
needs to be added to complete the branch, o/w the shell script won't complete